pyvista/core/grid.py

```diff
@@ -583,6 +583,7 @@ def __init__(
         dimensions=None,
         spacing=(1.0, 1.0, 1.0),
         origin=(0.0, 0.0, 0.0),
+        direction_matrix=None,
         deep=False,
```

Comment:
This is technically a breaking change, might be best to put the new keyword at the end.
```suggestion
        deep=False,
        direction_matrix=None,
```

Reply:
+1 not breaking change.